app/controllers/KeyboardMove.scala

```diff
@@ -0,0 +1,14 @@
+package controllers
+
+import views._
+import lila.app._
+
+final class KeyboardMove(
+    env: Env
+) extends LilaController(env) {
+
+  def help =
```

Comment:
Probably overkill to make a whole new controller here, but wasn't sure where else to stick it